### ISSUE TITLE
Ajouter la possibilité d'éditer soi-même un événement qu'on propose

### ISSUE THREAD

**Issue by MagicFab** [mlhamel/agendadulibre#43]
Lors de l'AGA de FACIL un des membres soulevait le besoin de pouvoir éditer soi-même un événement lorsqu'il y a des modifications à faire après sa publication.

Ceci impliquerait probablement une nouvelle demande d'approbation, et probablement autre chose dans le processus actuel. J'ouvre ce bug report pour suivi.

**Comment by mlhamel** [comment]
Effectivement. je penche vers un model ou la personne recevrait un lien unique qui lui permettrait d'éditer son événement et aussi une confirmation lorsque l'événement sera modéré.

**Comment by millette** [comment]
Une occasion d'utiliser les JSON Web Token (JWT) http://self-issued.info/docs/draft-ietf-oauth-json-web-token.html

Peut-etre que https://github.com/GetBlimp/django-rest-framework-jwt et https://github.com/jpadilla/django-jwt-auth seront utiles.
